examples/custom-esbuild/sanity-esbuild-app-esm/esbuild/define-text-by-option-plugin.cjs

```diff
@@ -0,0 +1,11 @@
+function defineTitleByOptionPlugin(pluginOptions) {
+  return {
+    name: 'define-title',
+    setup(build) {
+      const options = build.initialOptions;
+      options.define.titleByOption = JSON.stringify(pluginOptions.title);
```

Comment:
I just adopted this from the other examples. Why is there a `JSON.stringify`?

Reply:
I don't see any particular reason for that. @arturovt can you shed some light on it?